react/react1/week1/ToDoList/src/components/ToDoItem/ToDoItem.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./ToDoItem.css";
+
+const ToDoItem = ({ todo }) => {
```

Comment:
you're specifying which props are needed, which is a good way of making more robust components. good job